03-viz.Rmd

```diff
@@ -1,40 +1,528 @@
+and continents are coloured differently than other landmasses, making it quite clear that continents are the largest 7 landmasses.
+
+
+### The Old Faithful eruption / waiting time dataset
+This dataset contains measurements of the waiting time between eruptions and the subsequent eruption duration (in minutes). **Question:** Is there a 
+relationship between the waiting time before an eruption to the duration of the eruption? 
+
+```{r 03-data-faithful}
+# old faithful eruption time / wait time data
+head(faithful)
+```
+
+Here again we are investigating the relationship between two quantitative variables (waiting time and eruption time). But if you look at the output of the `head` function, you'll notice that neither of the columns are ordered. So in this case, let's start again with a scatter plot:
+
+```{r 03-data-faithful-scatter}
+faithful_scatter <- ggplot(faithful, aes(x = waiting, y = eruptions)) + geom_point()  
+```
+```{r 03-data-faithful-scatter-print, echo = FALSE, results = 'hide', message = FALSE, warning = FALSE}
+print(faithful_scatter)
+```
+
+We can see that the data tend to fall into two groups: one with a short waiting and eruption times, and one with long waiting and eruption 
+times. Note that in this case, there is no overplotting: the points are generally nicely visually separated, and the pattern they form is clear.
+In order to refine the visualization, we need only to add axis labels and make the font more readable:
+
+```{r 03-data-faithful-scatter-2}
+faithful_scatter <- ggplot(faithful, aes(x = waiting, y = eruptions)) + 
+                         geom_point() +
+                         labs(x = 'Waiting Time (mins)', y = 'Eruption Duration (mins)') +
+                         theme(text = element_text(size = 18))
+```
+```{r 03-data-faithful-scatter-2-print, echo = FALSE, results = 'hide', message = FALSE, warning = FALSE}
+print(faithful_scatter)
+```
+
+
+### The Michelson speed of light dataset
+This dataset contains measurements of the speed of light (in kilometres per second with 299,000 subtracted) from the year 1879 
+for 5 experiments, each with 20 consecutive runs. **Question:** Given what we know now about the speed of 
+light (299,792.458 kilometres per second), how accurate were each of the experiments?
+```{r 03-data-morley}
+# michelson morley experimental data
+head(morley)
+```
+
+In this experimental data, Michelson was trying to measure just a single quantitative number (the speed of light). The dataset
+contains many measurements of this single quantity. To tell how accurate the experiments were, we need to visualize the 
+distribution of the measurements. We can do this using a *histogram*. A histogram helps us visualize how a 
+particular variable is distributed in a dataset by separating the data into bins, and then using vertical bars 
+to show how many data points fell in each bin. To create a histogram in `ggplot2` we will use the `geom_histogram` geometric
```

Comment:
Are we sure `geom_histogram()` is an object? It might make one, but we "use" the `geom_histogram()` function (*I think).